src/material-experimental/mdc-slider/slider.ts

```diff
@@ -164,6 +164,7 @@ export class MatSliderThumb implements AfterViewInit {
       : this._slider.max;
     this._elementRef.nativeElement.min = `${min}`;
     this._elementRef.nativeElement.max = `${max}`;
+    this._elementRef.nativeElement.step = `${this._slider.step}`;
```

Comment:
nit: `this._elementRef.nativeElement` is being repeated in several places. Would it make sense to move it out into a variable?